dali/operators/numba_function/numba_func.cc

```diff
@@ -17,52 +17,47 @@
 
 namespace dali {
 
-DALI_SCHEMA(experimental__NumbaFunc)
-  .DocStr(R"code(Invokes a compiled Numba function passed as a pointer.
+DALI_SCHEMA(NumbaFunc)
```

Comment:
Just an idea:
Maybe we should move this out of experimental in a different PR, after this one is merged and a notebook is added. When we have the notebook we probably will be able to better evaluate the API. 
Such PR wouldn't add much work. Just mechanical renaming from `experimental.` to top-level

Reply:
I know that normally we aren't that cautious with releasing an operator but this one is a bit unusual one, I would say.

Reply:
@banasraf How can I add python wrapper to experimental package?